jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_workspace_view.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
create_workspace_view is modeled after create_sorting_view. It is less complex - only takes in feed_uri and workspace_name.